commit 14d02eb0e45f054e8087746b0cfc6fa6a338e13d (refs/changes/64/1464/5)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2017-02-24T01:53:43+00:00 (2 years, 7 months ago)
    
    CNS-194 update cueball and mname deps to latest
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/package.json b/package.json
index fbd7204..ea1f6ed 100644
--- a/package.json
+++ b/package.json
@@ -26,15 +26,15 @@
     "assert-plus": "0.1.5",
     "bunyan": "1.5.1",
     "changefeed": "1.2.0",
-    "cueball": "1.2.4",
+    "cueball": "2.1.1",
     "dashdash": "1.10.1",
     "deep-equal": "1.0.1",
     "ipaddr.js": "1.0.3",
     "jsprim": "1.2.2",
     "ldapjs": "1.0.0",
     "lru-cache": "2.7.0",
-    "mooremachine": "2.0.0",
-    "mname": "1.3.2",
+    "mooremachine": "2.1.0",
+    "mname": "1.3.4",
     "redis": "2.1.0",
     "restify": "4.0.3",
     "restify-clients": "1.1.1",
